ops/testing.py

```diff
@@ -2167,21 +2167,9 @@ def relation_remote_app_name(self, relation_id: int) -> Optional[str]:
         if relation_id not in self._relation_app_and_units:
             # Non-existent or dead relation
             return None
-        if 'relation_broken' in self._hook_is_running:
```

Comment:
is _hook_is_running used for anything else? or was this the use case and we can pull it out?

Reply:
It's used for bypassing access control on relation data as well.